### PR TITLE
Update DNS model in MalwareConfig to match MACO

### PR DESCRIPTION
**Pull request by cccs-rs**
Related to: https://github.com/CybercentreCanada/Maco/pull/35

### CODE REVIEW

File: assemblyline/odm/models/ontology/results/malware_config.py [hunk 1]
```diff
@@ -9,6 +9,11 @@
               "reverse_proxy", "rootkit", "scanner", "scareware", "spammer", "trojan", "virus", "wiper",
               "webshell", "worm"]
 CONNECTION_USAGE = ["c2", "upload", "download", "propagate", "tunnel", "other", "ransom", "decoy"]
+DNS_RECORD_TYPES = ["A", "AAAA", "AFSDB", "APL", "CAA", "CDNSKEY", "CDS", "CERT", "CNAME", "CSYNC", "DHCID", "DLV",
```
Comment (cccs-kevin) [inline assemblyline/odm/models/ontology/results/malware_config.py:12]: duplicate of https://github.com/CybercentreCanada/assemblyline-base/blob/9ed6eb059026ba781f9244ffc390ed38591ab28b/assemblyline/odm/models/ontology/results/network.py#L24

File: assemblyline/odm/models/ontology/ontology.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 from assemblyline.odm.models.ontology.filetypes import PE
 
 Classification = forge.get_classification()
-ODM_VERSION = "1.6"
+ODM_VERSION = "1.7"
```
Comment (cccs-kevin) [inline assemblyline/odm/models/ontology/ontology.py:8]: this is an opportunity to introduce a change log regarding the ontology. this was requested by someone in the past

**Review by cccs-kevin — CHANGES_REQUESTED** [review]

File: assemblyline/odm/models/ontology/ontology.py (hunk 2 above)
Comment (cccs-rs) [reply assemblyline/odm/models/ontology/ontology.py:8]: That will likely be something we pipeline during builds, similar to the auto-generated docs. I have an idea of how we want to do that and show obvious things like additions and removals that would be ODM-wide.

File: assemblyline/odm/models/ontology/results/malware_config.py [hunk 3]
```diff
@@ -11,6 +11,7 @@
 CONNECTION_USAGE = ["c2", "upload", "download", "propagate", "tunnel", "other", "ransom", "decoy"]
 
 
+
```
Comment (cccs-kevin) [inline assemblyline/odm/models/ontology/results/malware_config.py:14]: ✂️

File: assemblyline/odm/models/ontology/results/malware_config.py [hunk 4]
```diff
@@ -184,7 +187,7 @@ class Registry(odm.Model):
         odm.Enum(values=['persistence', 'store_data', 'store_payload', 'read', 'other']),
         description="Use of registry key")
 
-
+@odm.model(description="Extracted Malware Configuration")
```
Comment (cccs-kevin) [inline assemblyline/odm/models/ontology/results/malware_config.py:190]: I think we still want this newline

**Review by cccs-kevin — APPROVED** [review]

**Pull request merged**